swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger" : "2.0",
   "info" : {
     "description" : "VMware Cloud Foundation is an integrated software stack that bundles the VMware virtualization and management components like compute virtualization (vSphere), storage virtualization (vSAN), network virtualization (NSX), and cloud management (vRealize Suite). VCF consists of Cloud Builder - that automates the bringup of the management infrastructure and SDDC Manager - that automates the workload domain management, lifecycle and operations. This API reference guide covers APIs of both Cloud Builder and SDDC Manager",
-    "version" : "4.5.0",
+    "version" : "4.5.2-vcf4520RELEASE",
```

Comment:
This version string looks odd. Did it come from the release branch?

Reply:
It came from the recommended build (the same for some mouths now) from vcf4520. Maybe I should leave "4.5.2" only?